be/src/olap/in_list_predicate.h

```diff
@@ -346,6 +346,17 @@ class InListPredicateBase : public ColumnPredicate {
         }
     }
 
+    bool evaluate_and(const StringRef* dict_words, const size_t count) const override {
```

Comment:
In such scenaro:
1. the dict is [1,2,3]
2. the query condition is where a not in [1]
3. opposite = false
then
found = true and PT != IN LIST then the return value is false;
Its wrong